js/hview.js

```diff
@@ -557,20 +585,19 @@ $(function () {
         }
     }
 
-    const renderList = function (root, vListContent = $("#vlist_content").empty(), boxContent = $("#border-box").empty()) {
+    const renderList = function (root, vListContent = document.getElementById("vlist_content"), boxContent = document.getElementById("border-box")) {
         $("#hview").removeClass("hide").removeClass("hidden");
         $("#main-progress").hide();
```

Comment:
move these two outside as well, as they are updating the global properties.

Reply:
I will do that.